ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -736,14 +741,18 @@ module('Acceptance | allocation detail (services)', function (hooks) {
   });
 
   test('Allocation has a list of services with active checks', async function (assert) {
-    await visit('jobs/service-haver@default');
-    await click('.allocation-row');
+    const runningAlloc = server.db.allocations.findBy({
+      jobId: 'service-haver',
+      clientStatus: 'running',
+    });
+    await Allocation.visit({ id: runningAlloc.id });
```

Comment:
This test was flaky because the order of allocation could change, so the test could end up in an allocation that was not running.

Reply:
Extracted to https://github.com/hashicorp/nomad/pull/17648